test/lib/ansible_test/_data/setup/remote.sh

```diff
@@ -11,11 +11,8 @@ cd ~/
 install_pip () {
     if ! "${python_interpreter}" -m pip.__main__ --version --disable-pip-version-check 2>/dev/null; then
         case "${python_version}" in
```

Comment:
Might as well drop the `case/esac` here?

Reply:
I thought about that, but decided to leave it since we'll need it whenever we bump the pip bootstrapper version -- unless we've stopped supporting Python 2.7 by then.